### ISSUE TITLE
Zoom Breaks Lots of Stuff

### ISSUE THREAD

**Issue by wsloand** [roberttdev/dactyl4#11]
In #10, there is a discussion of zoom which breaks things.  The zoom discussion seems separate.

If we move to a single zoom level, I think it would need to load the largest image and auto-adjust to the width of the viewing window.  When I did some quick checks, with just editing the image css, it didn't work.

Generally, I think that due to differences in monitor sizes having different zoom levels is useful (so that it is possible to see the whole image regardless of monitor size) and to be able to distinguish what may be difficult to distinguish at the default zoom could also be useful.

So, in summary, I think that different zoom levels help, but that it may be possible to change the interaction to a percentage soom instead of using the current functionality.
